tests/ocean/test_compute_flow.py

```diff
@@ -17,6 +17,16 @@
     mint_tokens_and_wait, get_resource_path, wait_for_ddo)
 
 
+def test_check_url():
+    check_url_endpoint = DataServiceProvider.build_check_url_endpoint()
+    result = DataServiceProvider.check_url(
+        'http://xkcd.com/349/info.0.json', check_url_endpoint
+    )
+
+    assert result['contentLength'] == '629'
```

Comment:
```suggestion
    assert result['contentLength'] == '1161'
```

Reply:
not sure why the same file is 1161 bytes in s3, I'm assuming that when I saved your response somehow it was converted to UTF8, doubling the size.. :)